nexus-repository-p2/src/main/java/org/sonatype/nexus/repository/p2/rest/P2ProxyRepositoriesApiResourceBeta.java

```diff
@@ -22,9 +22,16 @@
 import org.sonatype.nexus.repository.rest.api.AuthorizingRepositoryManager;
 import org.sonatype.nexus.repository.rest.api.RepositoriesApiResourceBeta;
 
+import io.swagger.annotations.Api;
+
+/**
+ * @since 1.1.7
+ */
```

Comment:
please add here:  * @deprecated beta prefix is being phased out, prefer starting new APIs with {@link APIConstants#V1_API_PREFIX} instead